### PR TITLE
Change monoallelic and het flags to only flag variants that are not AC0

### PR DESCRIPTION
**Pull request by jkgoodrich**

### CODE REVIEW

File: gnomad_qc/v4/variant_qc/final_filter.py [hunk 1]
```diff
@@ -330,12 +330,11 @@ def generate_final_filter_ht(
...
     # Generate expressions for mono-allelic and only-het status if requested.
     if mono_allelic_flag:
-        vqc_expr = vqc_expr.annotate(
-            monoallelic=(raw_freq_expr.AF == 1) | (raw_freq_expr.AF == 0)
-        )
+        vqc_expr = vqc_expr.annotate(monoallelic=(raw_freq_expr.AF == 1))
     if only_het_flag:
         vqc_expr = vqc_expr.annotate(
-            only_het=((adj_freq_expr.AC * 2) == adj_freq_expr.AN)
+            only_het=(raw_freq_expr.AC > 0)
```
Comment (mike-w-wilson) [inline gnomad_qc/v4/variant_qc/final_filter.py:336]: Shouldnt this be `adj_freq_expr`? 
As is it still flags both mono and only_het for AC0 sites that are raw AF of 1 and homalt =1  sites when adj AC,AN , and homalt are 0

Comment (mike-w-wilson) [reply gnomad_qc/v4/variant_qc/final_filter.py:336]: Screenshot 2023-10-24 at 10 45 30 AM
These guys

Comment (jkgoodrich) [reply gnomad_qc/v4/variant_qc/final_filter.py:336]: oh yes, sorry about that, thank you

**Review by mike-w-wilson — APPROVED** [review]
:shipit:

**Pull request merged**